components/search-modal/element.js

```diff
@@ -13,6 +14,7 @@ export class MDNSearchModal extends L10nMixin(LitElement) {
   static styles = styles;
 
   static properties = {
+    _hasChanged: { state: true },
```

Comment:
This shouldn't be necessary: we're not re-rendering based on `_hasChanged`

Reply:
So just initialize in the constructor instead?

Reply:
No need: the existence of it here does no initialization, just tells Lit to look out for changes to it to re-render. With no initialization in the constructor, it defaults to undefined, which is falsy: hence why your code works. You can add it for completeness if you want.

Reply:
Fixed via [3c6eeef](https://github.com/mdn/fred/pull/795/commits/3c6eeef64218aadf75392e7f351015f8b84acc76), and renamed to `_hasEngaged` to make the intent clearer.